proposals/0317-sdl-protocol-security-specification.md

```diff
@@ -370,7 +264,115 @@ The following query header is used by the system and the application to send err
   </tr>
 </table>
 
-#### 4.7.5.4 Actions in case of an error
+
+## 7. Secured Communication
+
+It is possible to establish a secured and encrypted communication with the system by setting the frame header encryption flag to `1` when starting a new service. If the authentication was successful, the system will reply with a `StartService ACK` frame with the encryption flag also set to `1` indicating that encrypted data is now accepted. If the authentication fails for some reason the system will reset the TLS connection and return a `StartService NAK` frame.
+
+The RPC service needs to be started as unencrypted first, then moved to encrypted by sending another `StartService` at a later point. Other services can do the same thing to move from unencrypted to encrypted.
```

Comment:
Propose to change to: `.. then moved to encrypted state by sending another StartService request at ...`

Reply:
@dboltovskyi thanks, changed in https://github.com/LuxoftSDL/sdl_evolution/pull/7/commits/e463f69d7b47d7280aa2472ab9ab84bc27655b44